packages/react-heartwood-components/src/components/Text/Text.js

```diff
@@ -42,30 +95,36 @@ export type TextProps = {
 }
 
 const Text = (props: TextProps) => {
-	const { children, className, element, ...rest } = props
+	const { children, className, element, context, ...rest } = props
 	let Element = 'p'
+	let text = children
```

Comment:
`text` might not be text, like you see below; maybe store original children as `originalChildren`, then `let children = originalChildren` so you can override it. Makes it a little more compatible w/ the usual usages of `{children}`